frontend/sass/styles.scss

```diff
@@ -256,8 +259,18 @@ fieldset {
         background: $light;
     }
 
-    li > div.jf-biglist-counter + div {
-        margin-left: 1em;
+    li > div.jf-biglist-counter {
+        // Align counters to top of their container:
+        margin-bottom: auto;
```

Comment:
Tahnee wanted the biglist counters to be aligned to the tops of their containers, which will change their styling on all confirmation pages. Since we're adding a bit more text, I think that makes sense. Hope that is ok with you!